yumi_description/urdf/yumi.urdf

```diff
@@ -598,13 +520,9 @@
       <material name="Grey"/>
     </collision>
   </link>
-  <!--Extensions -->
-  <!-- body -->
-  <gazebo reference="yumi_body">
```

Comment:
Ya, I'm not sure if it affected anything. I tried running it with that code in there and without, and it seems as if it didn't matter as long as whichever color I gave it was in the [materials.xacro](https://github.com/ethz-asl/yumi/blob/addition/SetupWS/yumi_description/urdf/materials.xacro) file.